doc/api/vm.md

```diff
@@ -460,16 +464,10 @@ const contextifiedObject = vm.createContext({ secret: 42 });
 
   // Step 3
   //
-  // Evaluate the Module. The evaluate() method returns a Promise with a single
-  // property "result" that contains the result of the very last statement
-  // executed in the Module. In the case of `bar`, it is `s;`, which refers to
-  // the default export of the `foo` module, the `secret` we set in the
-  // beginning to 42.
+  // Evaluate the Module.
```

Comment:
For what it's worth I agree no comment is better than an incorrect comment but it would still be useful to correct rather than remove.